gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,17 +1,6 @@
-#
```

Comment:
I'm sure the tool removed this, but should we add this header back in?

Reply:
> Just for my own edification, did you run `./gradlew wrapper --gradle-version=7.4.1` to do the update? When I did that, it also made a lot of changes to `gradlew` but I don't see that file changed in this PR.

Yes, I did that but reverted wrapper script (we could update it but it seems like there are no issues with the current one).

Reply:
... and the header was wiped out by this update, I will put it back, thanks!

Reply:
@andrross fixed header, thanks!